build.sbt

```diff
@@ -88,6 +88,10 @@ val libraryapi  = "io.github.scalan" %% "library-api" % specialVersion
 val libraryimpl = "io.github.scalan" %% "library-impl" % specialVersion
 val libraryconf = "io.github.scalan" %% "library-conf" % specialVersion
 
+val circeVersion = "0.11.1"
+val circecore = "io.circe" %% "circe-core" % circeVersion
```

Comment:
Let's follow camelNamingStyle for local values

Reply:
i suggest to rename libraryimpl, libraryconf and libraryapi for unification then